strings.txt

```diff
@@ -24585,26 +24722,1098 @@
+    zh-Hans = 启动应用程序即表明同意“使用条款”和“隐私政策”。
+    zh-Hant = 啟動本應用程式，即表示您同意服務條款和隱私權原則。
+    el = Με την εκκίνηση αυτής της εφαρμογής, συμφωνείτε με τους όρους χρήσης και την πολιτική προστασίας προσωπικών δεδομένων.
+    sk = Spustením aplikácie súhlasíte s našimi podmienkami používania a zásadami ochrany osobných údajov.
+    sw = Kwa kuanzisha programu-tumizi hii, unakubaliana na Sheria za Utumizi na Sera ya Faragha.
+
+  [button_accept_and_continue]
+    en = Accept and continue
+    ru = Принять и продолжить
+    ar = قبول ومتابعة
+    cs = Přijmout a pokračovat
+    da = Accepter og fortsæt
+    nl = Aanvaarden en doorgaan
+    fi = Hyväksy ja jatka
+    fr = Accepter et continuer
+    de = Zustimmen und fortfahren
+    hu = Fogadja el, és folytassa
+    id = Terima dan lanjutkan
+    it = Accetta e continua
+    ja = 同意して続ける
+    ko = 동의하고 계속 진행
+    nb = Godta og fortsett
+    pl = Zaakceptuj i kontynuuj
+    pt = Aceitar e continuar
+    ro = Acceptare și continuare
+    es = Aceptar y continuar
+    sv = Godkänn och fortsätt
+    th = ตกลงและดำเนินการต่อไป
+    tr = Kabul et ve devam et
+    uk = Прийняти та продовжити
+    vi = Chấp nhận và tiếp tục
+    zh-Hans = 接受并继续
+    zh-Hant = 接受並繼續
+    el = Αποδοχή και συνέχεια
+    sk = Prijať a pokračovať
+    sw = Kubali na uendelee
+
+  [button_learn_more]
+    en = Learn more
+    ru = Подробнее
+    ar = تعلم المزيد
+    cs = Zjistit více
+    da = Flere oplysninger
+    nl = Meer informatie
+    fi = Lisätietoja
+    fr = En savoir plus
+    de = Weitere Informationen
+    hu = Tudjon meg többet
+    id = Pelajari selengkapnya
+    it = Ulteriori informazioni
+    ja = 詳細情報
+    ko = 자세히 알아보기
+    nb = Finn ut mer
+    pl = Dowiedz się więcej
+    pt = Saber mais
+    ro = Aflați mai multe
+    es = Leer más
+    sv = Läs mer
+    th = ศึกษาเพิ่มเติม
+    tr = Daha fazla bilgi edinin
+    uk = Докладніше
+    vi = Tìm hiểu thêm
+    zh-Hans = 了解更多
+    zh-Hant = 瞭解更多資訊
+    el = Μάθετε περισσότερα
+    sk = Zistiť viac
+    sw = Jua mengi zaidi
+
+  [onboarding_not_storage_permission_title]
+    en = Grant access
+    ru = Предоставьте доступ
+    ar = منح حق الوصول
+    cs = Povolit přístup
+    da = Giv adgang
+    nl = Toegang verlenen
+    fi = Myönnä käyttöoikeus
+    fr = Autoriser l'accès
+    de = Zugriff gewähren
+    hu = Hozzáférés biztosítása
+    id = Berikan akses
+    it = Concedi l'accesso
+    ja = アクセス権の付与
+    ko = 액세스 승인
+    nb = Gi tilgang
+    pl = Udziel dostępu
+    pt = Conceder acesso
+    ro = Acordați acces
+    es = Otorgar acceso
+    sv = Bevilja åtkomst
+    th = ให้สิทธิ์เข้าถึง
+    tr = Erişime izin ver
+    uk = Надати доступ
+    vi = Cho phép truy cập
+    zh-Hans = 授予访问权限
+    zh-Hant = 授與存取權
+    el = Εκχώρηση άδειας
+    sk = Prideliť prístup
+    sw = Ruhusu ufikiaji
+
+  [onboarding_not_storage_permission_message]
+    en = Change permissions settings in your device. Needs a storage access to use MAPS.ME.
```

Comment:
Вот это блок ужасен. Лучше так: "Please give MAPS.ME a permission to use the storage in your device settings. Otherwise it could not download any maps." И на русском: «Пожалуйста, зайдите в настройки вашего устройства и разрешите MAPS.ME обращаться к памяти. Иначе оно не сможет загружать карты».